### PR TITLE
[Merged by Bors] - feat: port LinearAlgebra.Dfinsupp

### PR DESCRIPTION
**Pull request by LukasMias**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by LukasMias** [comment]
Naming changes: theorems of the shape `Dfinsupp.mapRange.xyz` -> `Dfinsupp.MapRange.xyz`. That's how we do it, right? The according namespace is called `MapRange`, after all. Could have extracted that to a separate PR, I suppose, but it seems minor enough.

**Comment by jcommelin** [comment]
@LukasMias I think `mapRange` was correct. Note that it wasn't a `namespace` but only a `section` that was called `MapRange`. I think only this `section` name was wrong, and the rest was correct.

File: Mathlib/LinearAlgebra/Dfinsupp.lean [hunk 1]
```diff
@@ -15,28 +15,30 @@ import Mathlib.LinearAlgebra.Basis
...
-This file should try to mirror `linear_algebra.finsupp` where possible. The API of `finsupp` is
+This file should try to mirror `LinearAlgebra.Finsupp` where possible. The API of `Finsupp` is
 much more developed, but many lemmas in that file should be eligible to copy over.
 
 ## Tags
 
 function with finite support, module, linear algebra
 -/
 
+-- Porting note: TODO Erase this line.
```
Comment (jcommelin) [inline Mathlib/LinearAlgebra/Dfinsupp.lean:40]:
```suggestion
-- Porting note: TODO Erase this line. Workaround for lean4#2074.
```

**Review by jcommelin — COMMENTED** [review]
Please do a global `s/MapRange/mapRange/g`.
Otherwise, one comment and the rest lgtm

bors d+

**Comment by LukasMias** [comment]
> @LukasMias I think `mapRange` was correct. Note that it wasn't a `namespace` but only a `section` that was called `MapRange`. I think only this `section` name was wrong, and the rest was correct.

Ah, damn, I felt confirmed in my bias since most theorems of the shape "abc.def.ghi" have both the "abc" and the "def" part capitalized. But I understand better now. I'll make the fixes by tonight, if there's any hurry with this file then others are free to make the fixes on my behalf. Thank you!

**Pull request closed without merging**